src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -45,7 +45,7 @@ internal class RazorLanguageServerClient : ILanguageClient, ILanguageClientCusto
     private readonly LSPRequestInvoker _requestInvoker;
     private readonly ProjectConfigurationFilePathStore _projectConfigurationFilePathStore;
     private readonly RazorLanguageServerLogHubLoggerProviderFactory _logHubLoggerProviderFactory;
-    private readonly IOutputWindowLogger? _outputWindowLogger;
+    private readonly IEnumerable<ILogger> _loggers;
```

Comment:
Why do we make it an `IEnumerable<ILogger>` rather than an `ILogger`? From what I remember `Logger` itself in `Microsoft.Extensions.Logging` already has an array of Loggers that it loops over one-by-one to write call `LogX` on. (refer to [Logger.cs](https://source.dot.net/#Microsoft.Extensions.Logging/Logger.cs,0d7a40689bcdb5fa,references))

Reply:
It does, but if we export another implementation of `ILogger` we're not implicitly adding it to that class. That's if you use `ILoggerFactory` I believe.